docusaurus/docs/reactnative/ui-components/message_input.mdx

```diff
@@ -60,7 +60,7 @@ This component must be used as child of `Channel` component.
 ## Props
 
 :::note
-All the configuration for `MessageInput` can be done on [`Channel`](../core-components/channel.mdx) component. Additionally please take a look at [Guide Section](../guides/message_input.mdx) about customizing MessageInput UI
+All the configuration for `MessageInput` can be done on [`Channel`](../core-components/channel.mdx) component. Additionally please take a look at [Guide Section](../guides/message_input_customization.mdx) about customizing MessageInput UI
```

Comment:
Additionally, please take a look at the/our Guide Section on how to customize the MessageInput UI